docs/list_of_discussions.jl

```diff
@@ -0,0 +1,15 @@
+####
+#### Defines list of discussions
+####
+
+discussions =
+Any[
+"Home" => "Discussions/index.md",
+"Atmos" => Any[
+        "Discussions/Atmos/EDMFEquations.md",
+    ],
+"Ocean" => Any[],
+"Land" => Any[],
+"Common" => Any[],
+"Numerics" => Any[],
```

Comment:
Comment out empty pages until we have some content there so they don't render in HTML:

```suggestion
# "Ocean" => Any[],
# "Land" => Any[],
# "Common" => Any[],
# "Numerics" => Any[],
```